applications/fishing-map/src/features/map/Map.tsx

```diff
@@ -138,8 +131,7 @@ const getLegendLayers = (
 }
 
 const MapWrapper = (): React.ReactElement | null => {
-  const mapRef = useMapboxRef()
-  const mapInstance = useMapboxInstance()
+  const map = useMapInstance()
```

Comment:
Minor, maybe I'll fix it later but naming this `map` is confusing to me. 

What is `map`? 
- the Map component in Map.tsx?
- the Map component from RMGL?
- the Mapbox GL map instance?

I'd rather have
```
const mapboxGLInstance = useMapboxGLInstance()
```

Reply:
I like it, I'll keep it in mind for next PRs